main.go

```diff
@@ -38,17 +38,20 @@ const (
 )
 
 var (
-	port        uint
-	ethereumRPC string
+	port                   uint
+	ethereumRPC            string
+	vestingContractAddress string
+	deploymentHeightStr    string
 
-	vestingContractAddress = common.HexToAddress("0x23d1bFE8fA50a167816fBD79D7932577c06011f4")
 	pollEventRetryDuration = 14 * time.Second // pretty much how long it takes to build a block
 	pollNewState           = time.Minute      // no need to poll too often
 )
 
 func init() {
 	flag.UintVar(&port, "port", 1789, "port of the http server")
 	flag.StringVar(&ethereumRPC, "eth-rpc", "", "ethereum RPC address")
+	flag.StringVar(&vestingContractAddress, "vesting-contract-address", "", "vesting contract address")
+	flag.StringVar(&deploymentHeightStr, "deployment-height", "", "deployment block height")
```

Comment:
You can make it a uint from the beginning as well, instead of a string that you have to parse later.